src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngine.cs

```diff
@@ -385,13 +357,13 @@ public string GetAbsolutePath(string executingFilePath, string pagePath)
             ViewLocationCacheKey cacheKey)
         {
             // Only use the area view location formats if we have an area token.
-            var viewLocations = !string.IsNullOrEmpty(expanderContext.AreaName) ?
-                AreaViewLocationFormats :
-                ViewLocationFormats;
+            IEnumerable<string> viewLocations = !string.IsNullOrEmpty(expanderContext.AreaName) ?
```

Comment:
Never mind. This is for the [line 370](https://github.com/aspnet/Mvc/pull/4556/files#diff-557f4da6c8548e623754e6eaf781ea5dR370).